pkg/cri/server/container_create.go

```diff
@@ -186,7 +187,7 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	snapshotterOpt := snapshots.WithLabels(snapshots.FilterInheritedLabels(config.Annotations))
 	// Set snapshotter before any other options.
 	opts := []containerd.NewContainerOpts{
-		containerd.WithSnapshotter(c.config.ContainerdConfig.Snapshotter),
+		containerd.WithSnapshotter(c.runtimeSnapshotter(ociRuntime)),
```

Comment:
How does it work with `criService.snapshotStore`? In the following-up PRs, changes might be needed for some other APIs that use `snapshotStore` (e.g. ImageFsInfo,  ContainerStats, etc.)

Reply:
Good points.
* for ContainerStats this should be no change of behavior, the snapshot info should still be accurate for different snapshotters.
* For ImageFsInfo, we probably want to do something like the following. BTW, the device mapper snapshotter seems to be using `/var/lib/containerd/io.containerd.snapshotter.v1.devmapper/` as the mount point, which does not really mean anything, but seems to be reasonable to return snapshotter usages separately.
![image](https://user-images.githubusercontent.com/28563006/167075550-27566b4a-63f3-4c26-b3de-9d0d2e69374b.png)

I can address the ImageFsInfo in a following-up PR.